eng/Directory.Build.Data.props

```diff
@@ -53,6 +53,11 @@
     <CodeAnalysisRuleSet>$(RepoEngPath)\CodeAnalysis.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
+  <!-- StyleCop configuration -->
+  <ItemGroup>
+    <AdditionalFiles Include="$(RepoEngPath)\stylecop.json" Link="stylecop.json" />
```

Comment:
IMO, I'd forget `Link` and set `Visible="false"`. These don't need to show up in ever project.

Reply:
I also don't see this file checked in yet. Perhaps dependent on @chidozieononiwu pending PR?